integration-testing/README.md

```diff
@@ -110,6 +110,18 @@ $ ./run_tests.sh --collect-only
 $ ./run_tests.sh --collect-only  test/test_star_connected.py
 ```
 
+The test runs the [mypy](https://pypi.org/project/pytest-mypy/) static type checker on your source files as part of your Pytest test runs now.
```

Comment:
This will need to mention that it is not enabled by default (see my comment below).